tests/suites/hup/README.md

```diff
@@ -18,10 +18,10 @@ Test the transition between balenaOS releases via the `hostapp-update` mechanism
 # current tests
 
 * [smoke test](./tests/smoke.js): check if we can HUP successfully
+* [storage migration](./tests/storagemigration.js): check if we can migrate to overlayfs
```

Comment:
Now that the rollbacks are merged I should have added them here as well. Maybe include them in this commit since I forgot?